builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -666,11 +668,12 @@ def writeServerlessFile(config, environment_logical_id){
   sh "sed -i -- 's/\${file(deployment-env.yml):providerRuntime}/${config['providerRuntime']}/g' serverless.yml"
   sh "sed -i -- 's/\${file(deployment-env.yml):providerMemorySize}/${config['providerMemorySize']}/g' serverless.yml"
   sh "sed -i -- 's/\${file(deployment-env.yml):providerTimeout}/${config['providerTimeout']}/g' serverless.yml"
-  sh "sed -i -- 's/\${file(deployment-env.yml):securityGroupIds}/${config['securityGroupIds']}/g' serverless.yml"
-  sh "sed -i -- 's/\${file(deployment-env.yml):subnetIds}/${config['subnetIds'] }/g' serverless.yml"
   sh "sed -i -- '/#Start:resources/,/#End:resources/d' ./serverless.yml"
   sh "sed -i -- '/#Start:isScheduleEnabled/,/#End:isScheduleEnabled/d' ./serverless.yml"
 
+  if ((config['service'] in configLoader.JENKINS.API_SERVICES) && config['domain'] == 'jazz' && configLoader.JENKINS.DOCKERIZED == "true") {
```

Comment:
As per https://github.com/tmobile/jazz-installer/pull/520#discussion_r276339890, this should be `configLoader.JAZZ.VPC_SERVICES.APIS`

Flags should be boolean (at least we should follow this going forward), Can we get to `configLoader.JENKINS.DOCKERIZED == true` instead of `configLoader.JENKINS.DOCKERIZED == "true"`

Reply:
Done.